### PR TITLE
COM-19901: Add Google Analytics tracker

### PR DESCRIPTION
**Pull request by SylvainGuittard**
jira issue: https://jira.ez.no/browse/COM-19901

### CODE REVIEW

**Comment by SylvainGuittard** [comment]
ping @clash82 @damianz5

File: app/Resources/views/pagelayout_tracking_head.html.twig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- Google Analytics -->
+<script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (clash82) [inline app/Resources/views/pagelayout_tracking_head.html.twig]: please add `{% autoescape 'js' %}` before script starts

File: app/Resources/views/pagelayout_tracking_head.html.twig [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+                (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-303624-17', 'auto');
+    ga('send', 'pageview');
+
```
Comment (clash82) [inline app/Resources/views/pagelayout_tracking_head.html.twig]: you don't need this line, do you? :)

**Comment by damianz5** [comment]
+1

File: app/Resources/views/pagelayout_tracking_head.html.twig (hunk 2 above)
Comment (SylvainGuittard) [inline app/Resources/views/pagelayout_tracking_head.html.twig]: I think it helps to read the variables send to the tracker

Comment (clash82) [inline app/Resources/views/pagelayout_tracking_head.html.twig]: really? it's just an empty line :)

Comment (SylvainGuittard) [inline app/Resources/views/pagelayout_tracking_head.html.twig]: OK. I want to make you happy. I will remove the empty lines

**Comment by SylvainGuittard** [comment]
Ready for your +1 @clash82 ;)

**Comment by clash82** [comment]
Lovely +1

**Pull request merged**